### ISSUE TITLE
Does this support fido yubikey registration & authentication 

### ISSUE THREAD

**Issue by chunsfire** [DavBfr/flutter_u2f#2]
By looking into the code; in file u2f_webauthn_js.dart. It seems the asked use case is not supported.

authenticatorSelection: web.AuthenticatorSelectionCriteria(
          residentKey: 'discouraged',
          requireResidentKey: false,
          userVerification: 'discouraged',
          authenticatorAttachment: 'cross-platform',
)

What I am looking for is to add capability in my app that allows user to do fido registration with yubikey & authenticate verified push using FIDO yubikey?

**Comment by DavBfr** [comment]
Yes it supports Fido/YubiKey registration and authentication, that's the whole purpose.
